src/custom-elements/src/constants/constants.ts

```diff
@@ -50,7 +50,7 @@ export type DateFormats = typeof DATE_FORMATS[number];
 export const CHI_TIME_AUTO_SCROLL_DELAY = 50;
 export const CHI_TIME_SCROLL_ADJUSTMENT = 24;
 
-export const TIME_PICKER_FORMATS = ['12hours', '24hours'];
```

Comment:
change `['12hours', '24hours'] `to `['12hr', '24hr']`, because `12hours/24hours` are used nowhere in the project, while `['12hr', '24hr']` are used in a few places